docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -1151,6 +1151,51 @@ As a test annotated with `@QuarkusIntegrationTest` tests the result of the build
 These tests will **not** work if run in the same phase as `@QuarkusTest` as Quarkus has not yet created the final artifact.
 ====
 
+== Mixing `@QuarkusTest` with other type of tests
+
+It is not possible to execute a `@QuarkusTest` test with other types of tests in a single execution run (e.g. a Maven Surefire Plugin execution).
```

Comment:
That sentence alone reads as if it's not possible to mix with other, non-`@Quarkus*` tests. But as the next sentence is explicitely listing the conflicting `@Quarkus*` flavors, I think this is clear enough.

Reply:
I adjusted things a bit.

Reply:
Perfection!